stable_baselines3/common/policies.py

```diff
@@ -617,6 +617,63 @@ def __init__(self,
                                                    optimizer_kwargs)
 
 
+class ContinuousCritic(BasePolicy):
```

Comment:
I see the major difference to regular critics are 1) use of q-functions 2) multiple q-functions and 3) actions are fed in to get single q-value, rather than predicting all at once. I think these difference should be clarified in docs.

Reply:
Well by "regular critics" you mean "discrete actions critics" no :p?

>I think these difference should be clarified in docs.

I will do that ;)